Fix window buttons in case CSD when globalmenu is active


--- a/browser/base/content/browser.css
+++ b/browser/base/content/browser.css
@@ -334,5 +334,5 @@ toolbar[customizing] #whats-new-menu-button {
 %ifdef MENUBAR_CAN_AUTOHIDE
 #toolbar-menubar[autohide=true]:not([inactive]) + #TabsToolbar > .titlebar-buttonbox-container {
-  visibility: hidden;
+  visibility: visible;
 }
 %endif
